tools/stress-cluster/prepare.ps1

```diff
@@ -0,0 +1,3 @@
+$copyDirectory = Resolve-Path (Join-Path -Path $PSScriptRoot -ChildPath "../../eng/common/")
+
+Copy-Item -Path $copyDirectory -Destination $PSScriptRoot -Force -Recurse
```

Comment:
This should be copied to a sub-directory for build rather than `common`. Also, I would put the prepare and dockerfile not in the top level directory. Perhaps `cluster/kubernetes/stress-test-addons/images`?